pkg/cli/cli.go

```diff
@@ -134,16 +146,32 @@ func newHostCommand() *cobra.Command {
 	return host
 }
 
-func runCreateHostCommand(c *cobra.Command) error {
-	baseURL := buildBaseURL(c)
+func runSubCommand(c *cobra.Command, args []string, runner subCommandRunner) error {
+	proxyURL := c.InheritedFlags().Lookup(httpProxyFlag).Value.String()
+	// Handles http proxy
+	if proxyURL != "" {
+		if _, err := url.Parse(proxyURL); err != nil {
+			return err
+		}
+		os.Setenv("HTTP_PROXY", proxyURL)
```

Comment:
Any reason not to use https://pkg.go.dev/net/http#Transport to set the proxy?
An environment variable is no different than a global variable.

Reply:
good catch! used Transport.